server/grammar/YmlLexer.g4

```diff
@@ -97,24 +99,26 @@ COND_OR: '||';
 EQUAL_ASSIGNMENT: '=';
 MULTIVALUED_ASSIGNMENT: ':=';
 
-OPEN_GRANULE: BACKSLASH OPEN_PAR;
-CLOSE_GRANULE: BACKSLASH CLOSE_PAR;
+OPEN_GRANULE: BACKSLASH OPEN_PAR -> pushMode(TEXT_GRANULE_MODE);
 
-fragment MINUS: '-';
-fragment PLUS: '+';
-fragment DIVIDE: '/';
-fragment TIMES: '*';
+ADD: '+';
+DIV: '/';
+MUL: '*';
+SUB: '-';
+MOD: 'mod';
 
-OPERATOR: MINUS | PLUS | DIVIDE | TIMES;
+fragment NON_ZERO_DIGITS: [1-9];
+fragment DIGITS: '0' | NON_ZERO_DIGITS;
```

Comment:
LOL maybe `[0-9]` would have made your trees slightly cleaner. Isn’t that factorization just for the sake of factorization ? xD Do as you wish (and maybe you’ve got a good reason), but still.